dags/catalyst.py

```diff
@@ -0,0 +1,59 @@
+"""
+DAG to schedule generation of performance reports for recently completed nimbus experiments.
+
+See the [catalyst repository](https://github.com/mozilla/catalyst).
+
+*Triage notes*
+
+This app will perform some bigquery queries, and generate statistical reports based on that data which are
+then published to https://protosaur.dev/perf-reports/index.html.
+
+Occasionally, the app can fail if it cannot understand the nimbus experiment configuration properly, but most
+of these cases should have coverage by now.  Other issues that may occur is if the sample size is too low which
+may cause some problems during the statistical analysis.  I'm still looking for more examples of these situations
+to fix.
```

Comment:
Since you're saying that app execution failures will be fixed by the performance team, you probably don't need to include this part in the triage notes (unless I'm misinterpreting).